bodhi/server/models.py

```diff
@@ -14,6 +12,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
```

Comment:
There is still duplication here https://github.com/ankit01ojha/bodhi/blob/8f4a951731fc8a2b32439092ae3587a404a3e076/bodhi/server/models.py#L173-L187